.github/workflows/post_build_and_test.yml

```diff
@@ -0,0 +1,85 @@
+name: Post Build & Test
+
+on:
+  workflow_run:
+    types:
+      - completed
+    workflows:
+      - 'Build & Test'
+
+jobs:
+  # This runs Chromatic visual regression tests without exposing secrets to untrusted code from
+  # third-party PRs.
+  #
+  # For more details, see:
+  # https://github.blog/changelog/2021-02-19-github-actions-workflows-triggered-by-dependabot-prs-will-run-with-read-only-permissions/
+  # https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+  Test_visual_regression:
+    runs-on: ubuntu-latest
+    if: github.event.workflow_run.conclusion == 'success'
+    steps:
+      - name: Git clone repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Cache Node modules
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        run: |
+          yarn install --frozen-lockfile
+
+      # Based on
+      # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#using-data-from-the-triggering-workflow
+      - name: Download pre-built Storybook
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
```

Comment:
Is it worth putting this in an external seperate script and referencing it?

Reply:
Hmm, yes, it was a bit annoying editing that here. It looks like it's possible: https://github.com/actions/github-script#run-a-separate-file

Will give it a go :+1:

Reply:
Got this working – have pushed a fixup.